sdk/src/math/spotBalance.ts

```diff
@@ -382,35 +382,61 @@ export function calculateInterestRate(
 	delta = ZERO,
 	currentUtilization: BN = null
 ): BN {
+	// todo: ensure both a delta and current util aren't pass?
```

Comment:
?

Reply:
will silently ignore delta if `currentUtilization` is passed